### ISSUE TITLE
Move cards to a stack on the far right not possible

### ISSUE THREAD

**Issue by roboticsalign** [nextcloud/deck#846]
Dear community,

I observer an issue, that is limiting the productivity especially on touchscreen devices:

If you want to move a card from one stack to another, it works only, if the target stack is on the start screen. I you have a few more stacks and want to move it outside the startscreen, it does not work.

Do I use the App in a wrong way?

**To Reproduce**

1. create a few more stacks, than would fit to the initial touchscreen
2. two ways to observe the behavior:
- tap and hold a card and try to move it to the very far right stack - it wont scroll to the right.
- tap and hold a card from the very right stack and try to move it to one next to it - you cant move the card, but move the screen.

**Expected behavior**
I would expect 
1. scrolling the screen when it is possible and than 
2. fix the screen and move the card to place it exactly

**Client details:**
 - OS: IOS , Anroid
 - Browser: Safari, Chrome
 - Version any version since 6 months
 - Device: Ipad, Samsung Galaxy

<details>
<summary>Server details</summary>

**Operating system**: 
Ubuntu

**Web server:**
Apache 

**Database:**
Mysql

**PHP version:**
7

**Nextcloud version:** (see Nextcloud admin page)
14.0.4

**Where did you install Nextcloud from:**
nextcloud website

**Signing status:**

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.
```
No errors have been found.

**List of activated apps:**

```
Enabled:
  - accessibility: 1.0.1
  - activity: 2.7.0
  - audioplayer: 2.5.1
  - bruteforcesettings: 1.3.0
  - calendar: 1.6.4
  - camerarawpreviews: 0.6.3
  - checksum: 0.4.2
  - cloud_federation_api: 0.0.1
  - comments: 1.4.0
  - contacts: 2.1.8
  - dav: 1.6.0
  - deck: 0.5.2
  - federatedfilesharing: 1.4.0
  - federation: 1.4.0
  - files: 1.9.0
  - files_external: 1.5.0
  - files_markdown: 2.0.5
  - files_pdfviewer: 1.3.2
  - files_sharing: 1.6.2
  - files_texteditor: 2.6.0
  - files_trashbin: 1.4.1
  - files_versions: 1.7.1
  - files_videoplayer: 1.3.0
  - firstrunwizard: 2.3.0
  - gallery: 18.1.0
  - groupfolders: 2.0.2
  - logreader: 2.0.0
  - lookup_server_connector: 1.2.0
  - metadata: 0.8.0
  - nextcloud_announcements: 1.3.0
  - notes: 2.5.1
  - notifications: 2.2.1
  - oauth2: 1.2.1
  - ocsms: 2.1.1
  - password_policy: 1.4.0
  - polls: 0.9.5
  - previewgenerator: 2.0.0
  - provisioning_api: 1.4.0
  - rainloop: 6.0.2
  - serverinfo: 1.4.0
  - sharebymail: 1.4.0
  - socialsharing_email: 1.0.5
  - spreed: 4.0.2
  - support: 1.0.0
  - systemtags: 1.4.0
  - tasks: 0.9.8
  - theming: 1.5.0
  - twofactor_backupcodes: 1.3.1
  - updatenotification: 1.4.1
  - user_ldap: 1.4.0
  - workflowengine: 1.4.0
Disabled:
  - admin_audit
  - encryption
  - files_clipboard
  - keeweb
  - mail
  - survey_client
  - user_external
```

**Nextcloud configuration:**

```
{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "192.168.xxx",
            "xxx"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/xxx",
        "dbtype": "mysql",
        "version": "14.0.4.2",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "appstore.experimental.enabled": true,
        "ldapIgnoreNamingRules": false,
        "mail_smtpmode": "smtp",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "loglevel": 2,
        "maintenance": false,
        "updatechecker": false,
        "theme": "",
        "htaccess.RewriteBase": "\/",
        "ldapProviderFactory": "\\OCA\\User_LDAP\\LDAPProviderFactory",
        "updater.release.channel": "production",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpsecure": "ssl",
        "mail_smtpauth": 1,
        "mail_smtpport": "465",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "app.mail.imaplog.enabled": true,
        "mysql.utf8mb4": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "enabledPreviewProviders": [
            "OC\\Preview\\PNG",
            "OC\\Preview\\JPEG",
            "OC\\Preview\\GIF",
            "OC\\Preview\\BMP",
            "OC\\Preview\\XBitmap",
            "OC\\Preview\\MP3",
            "OC\\Preview\\TXT",
            "OC\\Preview\\MarkDown",
            "OC\\Preview\\PDF"
        ]
    }
}
```

**Are you using an external user-backend, if yes which one:** 
- ActiveDirectory

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

</details>

**Comment by Tost69** [comment]
I've the same issue on my iPad Pro. I tried also to load the webpage in desktop view but this won't work either.

**Comment by roboticsalign** [comment]
I updated recently to NC17 and Deck 8, but there is no change. Would be great to make use of Deck on mobile devices!

**Comment by jakobroehrl** [comment]
It's fixed with the version 1.0
Can't reproduce it with the 1.0.0-alpha1.
